app/assets/v2/js/pages/profile-trust.js

```diff
@@ -648,6 +647,114 @@ Vue.component('brightid-verify-modal', {
   }
 });
 
+Vue.component('duniter-verify-modal', {
+  delimiters: [ '[[', ']]' ],
+  data: function() {
+    return {
+      showValidation: false,
+      validationStep: 'validate-duniter',
+      validationError: '',
+      publicKey: ''
+    };
+  },
+  computed: {
+  },
+  props: {
+    showValidation: {
+      type: Boolean,
+      required: false,
+      'default': false
+    }
+  },
+  template: `<b-modal id="duniter-modal" @hide="dismissVerification()" :visible="showValidation" center hide-header hide-footer>
+                <template v-slot:default="{ hide }">
+                  <div class="mx-5 mt-5 mb-4 text-center">
+                    <div class="mb-3">
+                      <h1 class="font-bigger-4 font-weight-bold">Verify your Duniter account</h1>
+                      <p>
+                        Duniter is a free software for free currencies: a P2P, <a href="https://en.wikipedia.org/wiki/Web_of_trust">Web of Trust</a> & <a href="https://en.wikipedia.org/wiki/Social_credit">Universal Dividend system.</a>
+                        <p class="mb-4">
+                          <a href="https://duniter.org/en/introduction/">Learn more.</a>
+                        </p>
+                      </p>
+                      <div>
+                        <h2 class="font-bigger-4 font-weight-bold"> You need to have these requirements:</h2>
+                        <ol>
+                          <li>let's check if there is already a record with your account.</li>
+                          <li>if there is a link to your gitcoin account in your Duniter record </li>
```

Comment:
i think the user is going to need instructions about how to link their gitcoin account.. like how do i create a record that links my gitcoin/duniter accounts?

Reply:
Yes, I will set up a notion with step by step for you to add your Duniter account.

Reply:
https://docs.google.com/document/d/1XhtOELj_YnkkqNsI02ZJHoVh2EDz7L_KvxTNXeiF8Fw/edit?usp=sharing